src/main/java/com/wix/mysql/EmbeddedMysql.java

```diff
@@ -14,13 +14,15 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.locks.ReentrantLock;
 
 import static com.wix.mysql.config.MysqldConfig.SystemDefaults.SCHEMA;
 import static com.wix.mysql.utils.Utils.or;
 import static java.lang.String.format;
 
 public class EmbeddedMysql {
     private final static Logger logger = LoggerFactory.getLogger(EmbeddedMysql.class);
+    private static final ReentrantLock localRepository = new ReentrantLock();
```

Comment:
As I noted in PR message, this is a quick-win type of fix. For one I wanted unlock devs blocked by this issue. Regarding file-based locks - I think this should be in flapdoodle lib. Current impl is not intermingled with flapdoodle part much and pretty local to my glue, introducing file-based locks in my code would make it really akward.

So maybe eventually, but for now it's a nice poor-mans solution:)